tests/integration/kubernetes/confidential_common.sh

```diff
@@ -82,3 +82,28 @@ function is_confidential_hardware() {
 
 	return 1
 }
+
+function create_loop_device(){
+	local loop_file="${1:-/tmp/trusted-image-storage.img}"
+	cleanup_loop_device
```

Comment:
Should pass `loop_file` to `cleanup_loop_device`:  `cleanup_loop_device "$loop_file"`

Reply:
Fixed.